src/test/java/org/junitpioneer/jupiter/EnvironmentVariableUtilsTests.java

```diff
@@ -0,0 +1,59 @@
+/*
+ * Copyright 2015-2020 the original author or authors.
+ *
+ * All rights reserved. This program and the accompanying materials are
+ * made available under the terms of the Eclipse Public License v2.0 which
+ * accompanies this distribution and is available at
+ *
+ * http://www.eclipse.org/legal/epl-v20.html
+ */
+
+package org.junitpioneer.jupiter;
+
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.assertj.core.api.Assertions.assertThatCode;
+import static org.assertj.core.api.Assertions.assertThatThrownBy;
+
+import java.security.AccessControlException;
+import java.security.Policy;
+
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+
+@DisplayName("EnvironmentVariableUtils")
+@ClearEnvironmentVariable(key = "TEST")
+class EnvironmentVariableUtilsTests {
+
+	@Test
+	@SetSystemProperty(key = "java.security.policy", value = "file:src/test/resources/default-testing.policy")
+	void shouldThrowAccessControlExceptionWithDefaultSecurityManager() {
+		executeWithSecurityManager(() -> assertThatThrownBy(() -> EnvironmentVariableUtils.set("TEST", "TEST"))
+				.isInstanceOf(AccessControlException.class));
+	}
+
+	@Test
+	@SetSystemProperty(key = "java.security.policy", value = "file:src/test/resources/reflect-permission-testing.policy")
+	void shouldModifyEnvironmentVariableIfPermissionIsGiven() {
+		executeWithSecurityManager(() -> {
+			assertThatCode(() -> EnvironmentVariableUtils.set("TEST", "TEST")).doesNotThrowAnyException();
+			assertThat(System.getenv("TEST")).isEqualTo("TEST");
+		});
+	}
+
+	/*
+	 * This needs to be done during the execution of the test method and cannot be moved to setup/tear down
+	 * because junit uses reflection and the default SecurityManager prevents that.
+	 */
+	private void executeWithSecurityManager(Runnable runnable) {
+		Policy.getPolicy().refresh();
+		SecurityManager securityManager = System.getSecurityManager();
```

Comment:
I'd have named this variable `original`. Everywhere it is used it is clear that it's a `SecurityManager`, so naming it `securityManager` is redundant - however `System.setSecurityManager(original)` clearly communicates that we reset (and not modify) the `SecurityManager`.
This is just a sidenote, don't feel the need to modify the PR just for this.

Reply:
> I'd have named this variable `original`. Everywhere it is used it is clear that it's a `SecurityManager`, so naming it `securityManager` is redundant - however `System.setSecurityManager(original)` clearly communicates that we reset (and not modify) the `SecurityManager`.
> This is just a sidenote, don't feel the need to modify the PR just for this.

I would *not* rename the variable! It only seems redundant because of Java 8. With 10+, using var at this section it would not be clear what a variable with the name `original` holds.

Reply:
I mean, even with `var` it would look like this:
```
var original = System.getSecurityManager();
// ... other code ...
System.setSecurityManager(original);
```
Seems clear to me?